web/html/src/manager/content-management/shared/api/use-lifecycle-actions-api.ts

```diff
@@ -49,8 +49,13 @@ const useLifecycleActionsApi = (props: Props): returnUseProjectActionsApi => {
       setIsLoading(true);
 
       const apiUrl = getApiUrl(props.resource, props.nestedResource, id);
-      const networkMethod: NetworkMethod = networkAction[action] || networkAction["get"];
-      const networkRequest = networkMethod(apiUrl, JSON.stringify(actionBodyRequest), "application/json");
+
+      let networkRequest: ReturnType<NetworkMethod>;
+      if (action === "get" || !networkAction[action]) {
```

Comment:
Just to make sure we don't miss anything 100% : could it happen that action is capitalized instead (`GET` instead of `get`)? If so, I'd check for `action.toLowerCase() === "get"`, otherwise forget about this comment :)

Reply:
In that case we'd fall through to the second condition (`!networkAction["GET"] === true`), but just for sanity I add both that and a type check for the action parameter so that it only takes the correct keys. Good catch! 👍